buildlib/FindLDSymVer.cmake

```diff
@@ -27,15 +27,25 @@ else()
 endif()
 
 # And matching source, this also checks that .symver asm works
-check_c_source_compiles("
-void ibv_get_device_list_1(void);
-void ibv_get_device_list_1(void){}
-asm(\".symver ibv_get_device_list_1, ibv_get_device_list@IBVERBS_1.1\");
-void ibv_get_device_list_0(void);
-void ibv_get_device_list_0(void){}
-asm(\".symver ibv_get_device_list_0, ibv_get_device_list@@IBVERBS_1.0\");
-
-int main(int argc,const char *argv[]){return 0;}" _LDSYMVER_SUCCESS)
+if (HAVE_FUNC_ATTRIBUTE_SYMVER)
```

Comment:
Why do we need if-else here?
Isn't HAVE_FUNC_ATTRIBUTE_SYMVER already checked that symver work?

Reply:
ATTRIBUTE_SYMVER is checking if the C compiler attribute is supported, this is testing if the linker option works. We need to use the attribute version or this test fails on LTO linkers.